geesRecorderClient/Client/Pages/Attendance/Project.razor

```diff
@@ -64,4 +89,25 @@ else
         Console.WriteLine("Enrol");
         NavigationManager.NavigateTo($"/attendance/enrol/{Id}");
     }
+
+    public void MarkAttendance()
+    {
+        Console.WriteLine("MarkAttendance");
```

Comment:
It's not nice (most times) to log values to the console when deploying. Use them while debugging and remove when done.